### ISSUE TITLE
free surface elevation

### ISSUE THREAD

**Issue by Dylan0824** [capytaine/capytaine#252]
Dear all
  The capytaine calculate the free surface elevation under the diffraction and radiation.
  If i want to calculate the total free surface elevation, how should i do? Just sum them all?
  Thank you.

**Comment by mancellin** [comment]
Yes, you can take a linear combination of them.
If the wave amplitude is $A$, the RAO of dof $i$ is $a_i$ and $\xi$ is the free surface elevation, then the total free surface elevation is something like

$$ \xi(x) = A \xi_\mathrm{diff}(x) + A \sum_i a_i \xi_{\mathrm{rad},i}(x) $$

**Comment by Dylan0824** [comment]
> 
Dear 
Should the elevation of diffraction include the elevation of incident wave? Or just the elevation of diffraction.
Thank you .

**Comment by mancellin** [comment]
You're right, I forgot that. The free surface elevation computed for the diffraction problem in Capytaine does not include the elevation of the incident waves.

You can check the example in the documentation: https://github.com/capytaine/capytaine/blob/485ab2885e3856de21c0ed641751a44c2c0fd80b/docs/user_manual/examples/boat_animation.py#L34-L50

**Comment by Dylan0824** [comment]
> You're right, I forgot that. The free surface elevation computed for the diffraction problem in Capytaine does not include the elevation of the incident waves.
> 
> You can check the example in the documentation:
> 
> https://github.com/capytaine/capytaine/blob/485ab2885e3856de21c0ed641751a44c2c0fd80b/docs/user_manual/examples/boat_animation.py#L34-L50

Thank you. And i will check this.
